src/apps/explorer/ExplorerApp.tsx

```diff
@@ -30,28 +31,32 @@ const Home = React.lazy(
     ),
 )
 
-export const SwapAppV1: React.FC = () => {
+const HEADER = (
+  <Header
+    menu={
```

Comment:
well Dima's idea and also mine was to avoid using slotted components: `menu` kind of thing. But tbh I'm not really decided yet on what approach i like best. In a way this is nice because it forces a certain layout (which contrarians would state is wrong) but meh I kinda like it

Reply:
Agreed. That's the idea for his refactor, just the `GenericLayout` component that I'm not using it.
If we want to make it even more generic, we can indeed have it only accept children, and have another component called `Layout` that does have header/footer.

I can go that way if preferred.